js/views/baseVw.js

```diff
@@ -0,0 +1,86 @@
+import { View } from 'backbone';
+
+export default class BaseVw extends View {
+  constructor(options) {
+    super(options);
+    this._childViews = [];
+    this._unregisterFromParent = true;
+    this._removed = false;
+  }
+
+  /**
+   * If you are creating child views within your view, call this method
+   * to register them. This will ensure that they will have their remove
+   * method called if the parent is removed.
+   */
+  registerChild(childView) {
+    if (this._childViews.indexOf(childView) === -1) {
+      this._childViews.push(childView);
+      childView._parentView = this;
+    }
+  }
+
+  /**
+   * Opposite of registerChild. This method is automatically
+   * called by remove. For all practical purposes, you probably
+   * won't need to call this method directly.
+   */
+  unregisterChild(childView) {
+    const index = this._childViews.indexOf(childView);
+
+    if (index !== -1) {
+      this._childViews.splice(index, 1);
+      childView._parentView = null;
+    }
+  }
+
+  /**
+   * Shortcut method to instantiate a view and register it as a child.
+   * @param {function} ChildView - The class of the child view (not instance).
+   * @param {...*} args - Remaining arguments will be passed into the constuctor
+   *   when instantiating the child view.
+   * @return {object} The create child view instance.
+   */
+  createChild(ChildView, ...args) {
```

Comment:
This is an addition to the baseVw and in one shot you can create and register a child.

So, instead of:

```
const myChildView = new MyChildView();
this.registerChild(myChildView);
```

you can now:

```
this.createChild(MyChildView);
```